doc/protocols/scion-header.rst

```diff
@@ -473,3 +473,237 @@ Next Header
+that once observed or brute-forced the hop authenticators for some 
+path can use them to send arbitrary traffic along this path. EPIC-HP 
+solves this problem on the last link, which is particularly 
+important for the security of hidden paths.
+::
+
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                        PacketTimestamp                        |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                          PathMetaHdr                          |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                           InfoField                           |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                              ...                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                           InfoField                           |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                           HopField                            |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                              ...                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                           HopField                            |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                             PHVF                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                             LHVF                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+The EPIC-HP header has the following structure:
+   - A *PacketTimestamp* field (8 bytes)
+   - The path header of the SCION path type, where one bit 
+     of the Path Meta Header is used to indicate whether the sender 
+     accepts SCION path type response packets.
+   - A 4-byte *PHVF* (Penultimate Hop Validation Field)  and a 
+     4-byte *LHVF* (Last Hop Validation Field)
+
+The EPIC-HP header contains the full SCION path type header. The 
+calculation of the hop field MAC is identical. This allows the 
+destination host to directly send back (many) SCION path type answer 
+packets to the source. This can be done by extracting and reversing 
+the SCION path type header contained in the EPIC-HP packet.  
+
+This is allowed from a security perspective, because the SCION path 
+type answer packets do not leak information that would allow 
+unauthorized entities to use the hidden path. In particular, a SCION 
+path type response packet only contains strictly less information 
+than the previously received EPIC-HP packet, as the response packet 
+does not include the PacketTimestamp, the PHVF, and the LHVF.
+
+If the sender is behind a hidden path itself, then it is likely that 
```

Comment:
"behind hidden path", see comment in design document (also applies in the following).

Reply:
I changed it also in this document now.